### PR TITLE
Login page

### PR DESCRIPTION
**Pull request by FadiAlamassi**
relates #6

### CODE REVIEW

File: client/src/components/Pages/login/index.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+              htmlFor="email"
+              label="Email"
+              id="1"
+              type="text"
+              placeHolder="ex.fadi"
+              name="email"
+              value={account.email}
+              onChange={this.handleChange}
+            />
+            {error.email && <span className="errorMessage">{error.email}</span>}
```
Comment (AbdallahAmmar96) [inline client/src/components/Pages/login/index.js]: Use destruction for the error

File: client/src/components/Pages/login/index.js [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+        <div className="loginContainer">
+          <div className="loginContainer_input">
+            <h1 className="loginContainer__head">Log In</h1>
+            <LabeldInput
+              autoFocus
+              htmlFor="email"
+              label="Email"
+              id="1"
+              type="text"
+              placeHolder="ex.fadi"
```
Comment (AbdallahAmmar96) [inline client/src/components/Pages/login/index.js]:
```suggestion
              placeHolder="Ex. fadi"
```

Comment (AbdallahAmmar96) [inline client/src/components/Pages/login/index.js]: Is this a correct placeholder for email?
```suggestion
              placeHolder="Ex. fadi@gmail.com"
```

File: client/src/components/Pages/login/index.js [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+              value={account.email}
+              onChange={this.handleChange}
+            />
+            {error.email && <span className="errorMessage">{error.email}</span>}
+            <LabeldInput
+              htmlFor="password"
+              label="Password"
+              id="2"
+              type="password"
+              placeHolder="*********"
```
Comment (AbdallahAmmar96) [inline client/src/components/Pages/login/index.js]:
```suggestion
              placeHolder="********"
```

File: client/src/components/Pages/login/index.js [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+  };
+
+  handleSubmit = async event => {
+    event.preventDefault();
+    const { account: loginAccount } = this.state;
+    try {
+      await loginSchema.validate(loginAccount, {
+        abortEarly: false,
+      });
+      await axios.post('/api/v1/login', loginAccount);
```
Comment (AbdallahAmmar96) [inline client/src/components/Pages/login/index.js]: Show notification for the user with the message from the back end 
check the route /login or see the response

File: client/src/components/Pages/login/index.js [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+import axios from 'axios';
+import {
+  NotificationContainer,
+  NotificationManager,
+} from 'react-notifications';
+
+import loginSchema from '../../utils/validationSchemas/loginSchema';
+import LabeldInput from '../../common/LabeledInput';
+import Button from '../../common/Button';
+import 'react-notifications/lib/notifications.css';
```
Comment (AbdallahAmmar96) [inline client/src/components/Pages/login/index.js]: Put this import in the App/index.js 
so all team can use it

File: client/src/components/Pages/login/index.js [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+            />
+            {error.email && <span className="errorMessage">{error.email}</span>}
+            <LabeldInput
+              htmlFor="password"
+              label="Password"
+              id="2"
+              type="password"
+              placeHolder="*********"
+              name="password"
+              value={account.password}
```
Comment (AbdallahAmmar96) [inline client/src/components/Pages/login/index.js]: Use destruction for the account from the state

File: client/src/components/Pages/login/index.js [hunk 7]
```diff
@@ -0,0 +1,115 @@
...
+              placeHolder="********"
+              name="password"
+              value={passwordAccount}
+              onChange={this.handleChange}
+            />
+            {passwordError && (
+              <span className="errorMessage">{passwordError}</span>
+            )}
+            <p className="loginContainer__signup">
+              If you don't have an account{' '}
```
Comment (justinwiley) [inline client/src/components/Pages/login/index.js]: `{' '}` <- was this supposed to be a variable?  Maybe that is coming in a later PR?

File: client/src/components/utils/validationSchemas/loginSchema.js [hunk 8]
```diff
@@ -0,0 +1,14 @@
+import * as yup from 'yup';
+
+const loginSchema = yup.object({
+  email: yup
+    .string('Email should be string only')
+    .email('Email not vaild')
```
Comment (justinwiley) [inline client/src/components/utils/validationSchemas/loginSchema.js]: Email should be string only -> Email should be a string

vaild -> valid

File: client/src/components/Pages/login/index.js [hunk 9]
```diff
@@ -0,0 +1,115 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+import { NotificationManager } from 'react-notifications';
```
Comment (mossa-Sammer) [inline client/src/components/Pages/login/index.js]:
```suggestion
import { NotificationManager as notify } from 'react-notifications';
```

File: client/src/components/Pages/login/index.js [hunk 10]
```diff
@@ -0,0 +1,115 @@
...
+        NotificationManager.error(err.response.data.message);
+      } else {
+        const { history } = this.props;
+        history.push('/500');
+      }
+    }
+  };
+
+  handleChange = ({ currentTarget: input }) => {
+    const { account: loginAccount } = this.state;
```
Comment (mossa-Sammer) [inline client/src/components/Pages/login/index.js]:
```suggestion
    const {
     account: {...account}
     } = this.state;
```

File: client/src/components/Pages/login/index.js [hunk 11]
```diff
@@ -0,0 +1,115 @@
...
+    const account = { ...loginAccount };
```
Comment (mossa-Sammer) [inline client/src/components/Pages/login/index.js]:
```suggestion

```

File: client/src/components/Pages/login/index.js [hunk 12]
```diff
@@ -0,0 +1,115 @@
...
+        <div className="loginContainer">
+          <div className="loginContainer_input">
+            <h1 className="loginContainer__head">Log In</h1>
+            <LabeldInput
+              autoFocus
+              label="Email"
+              type="text"
+              placeHolder="Ex. fadi@gmail.com"
+              name="email"
+              value={emailAccount}
```
Comment (mossa-Sammer) [inline client/src/components/Pages/login/index.js]: you don't need value property here , you have the onChange method to update the state

File: client/src/components/Pages/login/index.js [hunk 13]
```diff
@@ -0,0 +1,115 @@
...
+              onChange={this.handleChange}
+            />
+            {emailError && <span className="errorMessage">{emailError}</span>}
+            <LabeldInput
+              label="Password"
+              type="password"
+              placeHolder="********"
+              name="password"
+              value={passwordAccount}
```
Comment (mossa-Sammer) [inline client/src/components/Pages/login/index.js]: you don't need value property here , you have the onChange method to update the state

**Pull request merged**